frontend/src/v5/ui/routes/login/passwordForgot/passwordForgot.component.tsx

```diff
@@ -34,7 +34,7 @@ export const PasswordForgot = (): JSX.Element => {
 		defaultValues: { username: '' },
 		resolver: yupResolver(PasswordForgotSchema),
 	});
-	const onSubmit = ({ username }) => API.Auth.resetPassword(username);
+	const onSubmit = () => API.Auth.resetPassword();
```

Comment:
I guess this whole component is not being used anymore?

Reply:
I removed the unused files, including this one, in https://github.com/3drepo/3drepo.io/pull/5456